smithy-cli/src/main/java/software/amazon/smithy/cli/dependencies/DependencyUtils.java

```diff
@@ -15,7 +15,9 @@
 import java.security.NoSuchAlgorithmException;
 
 final class DependencyUtils {
-    private DependencyUtils() {}
+    private DependencyUtils() {
+        // Utility Class should not be instantiated
```

Comment:
Nothing to do with this, and I know ppl ask for this, but I have never quite understood why "Utility Class should not be instantiated"

Reply:
I just add the comment so intellij and checkstyle wont throw any linting warnings for an empty method. 🤷